utils/src/test/scala/za/co/absa/enceladus/utils/config/UrisConnectionStringParserSuite.scala

```diff
@@ -13,39 +13,40 @@
  * limitations under the License.
  */
 
-package za.co.absa.enceladus.dao.rest
+package za.co.absa.enceladus.utils.config
 
-import za.co.absa.enceladus.dao.DaoException
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
 
-class RestApiConnectionStringParserSuite extends BaseTestSuite {
+class UrisConnectionStringParserSuite extends AnyWordSpec with Matchers {
```

Comment:
so you basically eliminated `MockitoSugar` and `BeforeAndAfter` traits. Why? (just curious, were they redundant?)

Reply:
Sorry didn't see this comment before. It is because this class has been moved to different package, and as only a part of what BaseTestSuite provides was needed, I decided to use only that.